src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs

```diff
@@ -132,8 +132,10 @@ public void Dispose()
 		Player?.Pause();
 	}
 
-	protected virtual partial ValueTask PlatformSeek(TimeSpan position)
+	protected virtual partial ValueTask PlatformSeek(TimeSpan position, CancellationToken token)
 	{
+		token.ThrowIfCancellationRequested();
```

Comment:
I can remember from some discussion that we didn't throw when the code is related to UI code, since there's no way to `try/catch` inside xaml... And I believe it's the case here (inside this manager), the user can't `try/catch` this, so the app could crash.

Reply:
That's fair. 

In regards to this code snipper, I don't think `Seek` is a UI thing, though. Looking at the docs, the `Position` property is read-only, and the only way to change it is in code via the `SeekTo()` method.

<img width="791" alt="image" src="https://github.com/CommunityToolkit/Maui/assets/13558917/f904260b-e7e9-4fab-a200-4d9de2cbfdaf">

Reply:
This isn't affected when the user moves that _video timeline_? I thought it can be changed by user interaction somehow

Reply:
I’ll dig a bit deeper and run some tests to confirm 👍 

I’ll plan on updating the sample app to include a “Skip Ahead 5 Seconds” button, and I’ll debug down into the code to ensure that we cannot trigger this via the UI.

Reply:
Good news! I confirmed that this code cannot be triggered by the UI, so we're all good!